R/virtualenv.R

```diff
@@ -180,6 +182,18 @@ virtualenv_exists <- function(envname) {
 
 virtualenv_path <- function(envname) {
 
+  # if envname is NULL but Python is already active, then
+  # use the path to the current virtual environment
+  if (is.null(envname) && py_available()) {
+    config <- py_config()
```

Comment:
This again violates the idea that installation functions shouldn't load Python (especially problematic on Windows where there can be DLL conflicts). I don't think py_install should key off of what's currently loaded, rather it should see whether it's running inside an renv.

Reply:
Sorry, that makes sense -- I've made that change.